lib/batch.js

```diff
@@ -68,6 +68,23 @@ module.exports = function(config) {
         });
     };
 
+    batch.get = function(ids, dataset, callback) {
+        var results= []
+        ids.forEach(function(id){
+            var key = { dataset: dataset, id: 'id!' + primary };
+            config.dyno.getItem(key, function(err, item) {
```

Comment:
Is any of this asynchronous? if so, the callback might fire before all `results` are compiled